docs/performance/quantization.md

```diff
@@ -48,18 +48,112 @@ The picture below shows the equivalent representation with the QOperator and QDQ
 ![Changes to nodes from basic and extended optimizations](../../images/QDQ_Format.png)
 
 ## Quantizing an ONNX model
-There are two ways of quantizing a model: dynamic and static.
 
-* **Dynamic quantization**: This method calculates the quantization parameters (scale and zero point) for activations dynamically.
+ONNX Runtime provides python APIs for converting 32-bit floating point model to an 8-bit integer
+model, a.k.a. quantization. These APIs include pre-processing, dynamic/static quantization, and
+debugging.
 
-* **Static quantization**: leverages calibration data to calculate the quantization parameters of activations. Our quantization tool supports three calibration methods: MinMax, Entropy and Percentile. Please refer to [`calibrate.py`](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/calibrate.py) for details.
+### Pre-processing
+{: .no_toc}
+
+Pre-processing is to transform a float32 model to prepare it for quantization. It consists
+of the following three optional steps:
+
+1. Symbolic shape inference. This is best suited for transformer models.
```

Comment:
Should these run in order? The numbered list makes it seem so.  If not, maybe change to a bulleted list